app/services/idv/profile_maker.rb

```diff
@@ -8,11 +8,12 @@ def initialize(applicant:, user:, user_password:)
       self.user_password = user_password
     end
 
-    def save_profile
-      profile = Profile.new(user: user, active: false)
+    def save_profile(active:, deactivation_reason: nil)
+      profile = Profile.new(user: user, active: false, deactivation_reason: deactivation_reason)
```

Comment:
Note: This must be `active: false` and not `active: active`, since the `activate` call below would [try to find an existing profile with `active: true`](https://github.com/18F/identity-idp/blob/e59299ec28510e8c60f672e3b0d778a9748772cb/app/models/profile.rb#L26) and incorrectly catch the profile which had just been created here.